watson/cli.py

```diff
@@ -374,6 +376,9 @@ def report(watson, current, from_, to, projects,
     `--tag` options. They can be specified several times each to add multiple
     projects or tags to the report.
 
+    If you are outputting to the terminal, you can selectively enable a pager
+    through the `--pager`` option.
```

Comment:
Here's the source of the extra backtick.

Reply:
fixed!